lib/matplotlib/backends/_backend_tk.py

```diff
@@ -658,7 +658,10 @@ def _set_image_for_button(self, button):
 
     def _Button(self, text, image_file, toggle, command):
         if not toggle:
-            b = tk.Button(master=self, text=text, command=command)
+            b = tk.Button(
+                master=self, text=text, command=command,
+                relief="flat", overrelief="groove", borderwidth=1,
```

Comment:
Is the border width inside or outside the button? Just wondering if the size we allocate is consistent on HiDPI screens?

Reply:
It's outside of the button, and the default width seems to be 2, so space shouldn't be an issue.

Reply:
I'm more concerned about the height of the toolbar than the width of the border itself.

Reply:
The toolbar height is set to the height of two text lines [here](https://github.com/matplotlib/matplotlib/blob/5555daa96aa9b4b57a652147d1dab53798272c1f/lib/matplotlib/backends/_backend_tk.py#L551), which on all platforms I've tried is bigger than the size of the buttons, so the buttons becoming 2px smaller shouldn't affect the height of the toolbar.